docs/semgrep-appsec-platform/jira.md

```diff
@@ -98,22 +120,28 @@ You can create tickets for Code findings using the **Triage** button on the:
 
 To create tickets:
 
+<!-- vale off-->
 1. If you're on the [**Findings**](https://semgrep.dev/orgs/-/findings) page, select the findings for which you want tickets created; you can select and create tickets for individual findings or all findings for a given rule. Otherwise, proceed to step 2.
-2. Click **Triage**.
-3. Set the status to **Fixing**, and select the **Create <span className="placeholder">NUMBER_OF_TICKETS</span> JIRA tickets in <span className="placeholder">PROJECT_NAME</span>** box that appears.
-4. Click **Change status** to proceed.
+1. Click **Triage**.
+1. Set the status to **Open**, **Fixing**, or **Reviewing**. Select **Fixing** if it is a known issue that needs to be fixed or **Reviewing** if the finding needs more investigation.
+1. Select the **Create tickets...** checkbox.
+    1. Optional: Click the first drop-down list to choose between making a ticket for **groups of findings** or an individual ticket for **each finding**.
+    1. Optional: Click the **JIRA project** drop-down list to select which Jira project to add the findings to.
+1. Optional: You can add **Comments** in the text box.
+1. Click **Submit** to proceed.
+<!-- vale on -->
 
 :::note
 Creating tickets for many findings at once may take some time. Tickets that take longer than 10 seconds to create are shown in Semgrep once you refresh the page.
```

Comment:
Possibly worth adding a note that if ticket creation fails, Semgrep will automatically retry several times over the next day to provide robustness against outages and down time in 3rd-party-services.

Reply:
![image](https://github.com/semgrep/semgrep-docs/assets/20766840/94b3a29a-7c07-472d-8a3d-93123ebba5fe)